src/main/webapp/WEB-INF/templates/user_dashboard.jsp

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html>
+<head>
```

Comment:
I think it'd be nice to have the Snippet logo on this page as well? It feels pretty empty.